content/en/blog/_posts/2021-04-12-introducing-suspended-jobs.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Introducing Suspended Jobs"
+date: 2021-04-12
+slug: introducing-suspended-jobs
```

Comment:
```suggestion
title: "Introducing Suspended Jobs"
￼date: 2021-04-12
￼slug: introducing-suspended-jobs
layout: blog
```

I think the blog layout is something that Hugo can infer, but for now I'd prefer to state it explicitly.

Reply:
did you mean to indent the date and slug fields with a space at the beginning?

Reply:
Indent? I don't see an indent.

Reply:
`layout: blog` is the bit I'd add.

Reply:
weird, I see some whitespace before the slug field in your suggested change. maybe github is acting up.

anyway, updated, thanks.

Reply:
If you copy and paste from GitHub's web UI rendering of the suggestion, there's a space at the start of each line. Maybe it's that?

Reply:
it's in the UI for me haha: [screenshot](https://user-images.githubusercontent.com/7521600/113286717-33073b00-930a-11eb-9c4e-52397abc2f1a.png)

Reply:
OK, well I'm happy to drop any unexpected Unicode - I didn't press the U+FFFC button when I typed out the suggestion!